src/Dropdown/Dropdown.tsx

```diff
@@ -31,10 +47,19 @@ const Dropdown: React.FC<DropdownProps> = ({
     onSelect,
     disabled = false,
     defaultIndex = 0,
+    selectedItem = items[defaultIndex],
```

Comment:
It isn't very clear that this is simply just the first selected item. I.e. "defaultItem"

You could consider removing the defaultIndex and only allow a defaultItem even though it breaks API.

Reply:
Yes I keep it there not to break other apps + I think if one it not interested in external updates then they can just use that. but I know this creates a paradox if both a set.... so not sure how to proceed.

Reply:
Yeah, I think it's fair to force the usage of one. I remember wanting to create a default based on the item but the index was more convenient at the time.
Item is more explicit and verbose though so I'd prefer that.